packages/ra-ui-materialui/src/list/ListNoResults.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { memo } from 'react';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import { useTranslate } from 'ra-core';
+
+export const ListNoResults = memo(() => {
+    const translate = useTranslate();
+    return (
+        <CardContent>
+            <Typography variant="body2">
+                {translate('ra.navigation.no_results')}
```

Comment:
You could pass the resource as parameter to the translation key here, which would help for customization of the label